config/constants/settings/definition.rb

```diff
@@ -923,6 +923,12 @@ class Definition
         default: "https://releases.openproject.com/v1/check.svg",
         writable: false
       },
+      seed_admin_user_enabled: {
+        description: "Enable setting the admin user on startup. " \
+                     "If you set this to false, You will have to create an admin user manually.",
```

Comment:
```suggestion
                     "If you set this to false, you will have to create an admin user manually.",
```

Reply:
Actually, let's avoid talking to the user
```suggestion
                     "If set to false, an admin user has to be created manually.",
```